packages/amplication-server/src/core/assistant/assistant.service.ts

```diff
@@ -186,14 +214,7 @@ export class AssistantService {
     threadId: string,
     context: AssistantContext
   ): Promise<AssistantThread> {
-    if (!this.assistantFeatureEnabled)
-      throw new AmplicationError("The assistant AI feature is disabled");
-
-    if (context.user.workspace.allowLLMFeatures === false) {
```

Comment:
You deleted this check (allowLLMFeatures) from the server side and left it only on the client side. Is that intentional?

Reply:
it was moved to a common function and used from 2 places in the server side
look on the new code inthis PR

Reply:
In this PR there are 3 places that allowLLMFeatures appears, two in the client  and one here where it was removed from the server. The other check for assistantFeatureEnabled is indeed in the common function but this check isn't.

Reply:
LOL, GH diff view fail. Just a folded part of the function that I didn't see...